spec/tree-view-spec.coffee

```diff
@@ -3007,6 +3007,70 @@ describe "TreeView", ->
         expect(atom.notifications.getNotifications()[0].getMessage()).toContain 'Opening folder in Finder failed'
         expect(atom.notifications.getNotifications()[0].getDetail()).toContain 'ENOENT'
 
+  describe "showCurrentFileInFileManager()", ->
+    it "does nothing when no file is opened", ->
+      expect(atom.workspace.getPaneItems().length).toBe(0)
+      expect(treeView.showCurrentFileInFileManager()).toBeUndefined()
+
+    it "does nothing when only an untitled tab is opened", ->
+      waitsForPromise ->
+        atom.workspace.open()
+      runs ->
+        $(workspaceElement).focus()
+        expect(treeView.showCurrentFileInFileManager()).toBeUndefined()
+
+    describe "showCurrentFileInFileManager() when some file is opened", ->
```

Comment:
> I think these tests can be removed.

do you mean _these part_?   > code block inside this function:

`describe "showCurrentFileInFileManager() when some file is opened", ->`

(line 3022~3072)

P.S. My opinion to the removal of this part is neutral.
The only diffs of these test (compared with original) are _Notification message_ assertion
( replace 'folder' with 'file' )